Makefile.standalone

```diff
@@ -0,0 +1,52 @@
+
+## path configuration (FHS-style)
+PREFIX ?= /usr
+BINDIR ?= $(PREFIX)/bin
+SBINDIR ?= $(PREFIX)/sbin
+
+## toolchain commands
+export GO ?= go
+export INSTALL ?= install
+export LN_S ?= ln -s
+export MKDIR_P ?= mkdir -p
+
+BUILD_TMP ?= $(CURDIR)/tmp
+BUILD_OUT ?= $(BUILD_TMP)/build
+export GOCACHE ?= $(BUILD_TMP)/go-cache
+export LDFLAGS=-s -w
+
+BUILD_GO_ARCH := $(shell $(GO) env GOARCH | sed -e 's~^386~i386~;')
+BUILD_GO_OS := $(shell $(GO) env GOOS)
+
+EXEC_NAME=dockerd-$(BUILD_GO_OS)-$(BUILD_GO_ARCH)
+EXEC_OUT=$(BUILD_OUT)/$(EXEC_NAME)
+
+# make sure it fails ...
+export GOPATH=/proc/gopath
+export GOPROXY=127.0.0.1:29999
+
+all: compile
+
+compile: $(EXEC_OUT)
+
+## generate go.mod in order to use simpler module build mode
```

Comment:
You shouldn't need go.mod (for the current version of Go).
Just GO111MODULE=off should work.

Reply:
Unfortunately, doesn't work. Had to take out -mod=vendor (which doesnt work of modules are switched off),
now I'm getting: 

cannot use path@version syntax in GOPATH mode

Reply:
I've spent several hours to get it compiling, the current approach is the only way I could it get working.